adoc/task_list.1.adoc

```diff
@@ -99,4 +99,14 @@ $ globus task list --limit 100 \
     --filter-not-label 'autolabel' --exact
 ----
 
+List active transfers in a tabular format suitable for consumption by unix
+tools:
+
+----
+$ globus task list --format unix \
+    --jmespath 'DATA[?status==`ACTIVE`].[task_id, source_endpoint_id, destination_endpoint_id, label]'
+----
+
+NOTE: 'destination_endpoint_id' will be 'null' in the case of Delete tasks.
```

Comment:
Well, if would be `null` for `--format=json`, but you have `--format=unix` in this example, so it'll actually be `None`.

Reply:
Yes, thank you. Should I document as `None`, or maybe just drop the note and people can learn by trying it out?